### PR TITLE
Add initial schematron.

### PR DESCRIPTION
**Pull request by ohsh6o**
This is from experimentations and our pairing sessions.

### CODE REVIEW

**Review by mike-stern — CHANGES_REQUESTED** [review]
Really great work to pull all the requisite information in, traverse it & provide access to appropriate profiles. I think we'll work on finding options for reducing the amount of a priore knowledge required, but this is a great start.

Thanks for adding great tests. I think there should be spec scenarios for 
- `no-security-sensitivity-level` 
-  `incomplete-implementation-requirements`

**Comment by ohsh6o** [comment]
It does not seem like I can get the global-context-item to work. I am attaching a screenshot here to ease communication with XSpec user group mailing list.

xspec-error

**Comment by ohsh6o** [comment]
@brianrufgsa we touched up the shell scripts in this branch along with the XPath modifications we discussed. Are you ready to re-review and potentially approve the basic test layout? We updated the docs. Also, @mike-stern, I will leave this to formal review, including your changes from earlier today. :-)

**Review by mike-stern — APPROVED** [review]
left a couple comments, but none that should block the merge, at your leisure.

File: resources/validations/src/ssp.sch
Comment (mike-stern) [reply resources/validations/src/ssp.sch:83]: locally, i had a merge conflict (simple resolution) when coming from master:HEAD, not sure if you want me to push that merge commit or not

File: resources/validations/src/ssp.sch
Comment (ohsh6o) [reply resources/validations/src/ssp.sch:69]: I actually had a discussion with either Wendell or Brian and they suggested it really is better practice, if not stylistically more consistent. The other place below where I made it more lax was similarly recommended. We do not want to adhere to people who do not properly configure the `<system-security-plan>` element nested in the wrong element. 

We might have been doing it with either approach but this make it more clear, per others.

File: resources/validations/src/ssp.sch
Comment (ohsh6o) [reply resources/validations/src/ssp.sch:83]: So I left this test in half-baked state because I need to debug it further. In short, something with the XSpec global-context-item pattern is not working and this test seems to work purely in Schematron, but fails due to missing context info in XSpec. I am waiting for feedback [from ther XSpec Google Group](https://groups.google.com/g/xspec-users/c/dNvDGZfKVz0). You will observe for now in [the Github Action XSpec results](https://github.com/18F/fedramp-automation/pull/4/checks?check_run_id=1394335757) it is 💤 (hidden/pending) and I marked testing around it that was for now to continue this later in #26 with other stuff once I work this out.

Comment (mike-stern) [reply resources/validations/src/ssp.sch:83]: makes sense

**Pull request merged**
